packages/sui-studio/src/components/style/index.js

```diff
@@ -19,8 +19,12 @@ export default class Style extends Component {
     this._linkElement = createLinkElement()
   }
 
+  componentWillUnmount () {
+    const lastIndex = document.styleSheets.length - 1
+    document.styleSheets[lastIndex].disabled = true
+  }
```

Comment:
we are supposing that it is always the last, I right now it is. But that can change.

I mean, for me the PR it is OK. That will work for sure, the only thing is that with a ID, you will know always exactly with node is, without import it other element in the studio id add other styles tag too and changing the order.